traefik/values.yaml

```
@@ -5,6 +5,27 @@ image:
   tag: ""
   pullPolicy: IfNotPresent
 
+#
+# Configure integration with Traefik Hub
+#
+hub:
+  ## Enabling Hub will:
+  # * enable Traefik Hub integration on Traefik
+  # * add `traefikhub-tunl` endpoint
+  # * enable addRoutersLabels on prometheus metrics
+  # * enable allowExternalNameServices on KubernetesIngress provider
+  # * enable allowCrossNamespace on KubernetesCRD provider
+  # * add an internal (ClusterIP) Service, dedicated for Traefik Hub
+  enabled: false
+  ## Default port can be changed
+  # traefikhubTunl: 9901
```

Maybe something like this is more clear?
```suggestion
  # tunnelPort: 9901
```

---

Yes it is !
=> PR updated accordingly.